editor/plugins/animation_tree_editor_plugin.cpp

```diff
@@ -419,6 +423,7 @@ void AnimationTreeEditor::_popup_edit_dialog() {
 				edit_label[1]->show();
 				edit_scroll[1]->set_min(0);
 				edit_scroll[1]->set_max(1);
+				edit_scroll[0]->set_step(0.01);
```

Comment:
I'm sorry for fuss, but I closed this pr and opened a new one here: https://github.com/godotengine/godot/pull/9749
This pr would push 2 commits into master and Akien would not be happy about that...